config/v1/types_cluster_version.go

```diff
@@ -278,6 +290,12 @@ const (
 // ClusterVersionCapabilitySets defines sets of cluster version capabilities.
 var ClusterVersionCapabilitySets = map[ClusterVersionCapabilitySet][]ClusterVersionCapability{
 	ClusterVersionCapabilitySetNone: {},
+	ClusterVersionCapabilitySet4_12: {
+		ClusterVersionCapabilityOpenShiftSamples,
+		ClusterVersionCapabilityBaremetal,
+		ClusterVersionCapabilityInsights,
```

Comment:
probably want to put yourself in `vCurrent` too, so you keep getting installed by default, and require explicit admin config for installs that exclude you.